bundles/org.openhab.binding.ojelectronics/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -22,10 +22,10 @@
 			</parameter>
 			<parameter name="apiUrl" type="text" required="true">
 				<label>API-URL</label>
-				<description>URL to cloud API-service.</description>
+				<description>URL to cloud API-service and Socket-Notification.</description>
 				<context>url</context>
 				<advanced>true</advanced>
-				<default>https://OWD5-OJ001-App.ojelectronics.com/api</default>
+				<default>https://OWD5-OJ001-App.ojelectronics.com</default>
 			</parameter>
 			<parameter name="refreshDelayInSeconds" type="integer" required="true" min="15" unit="s">
```

Comment:
This parameter is described in documentation, I just verified. So to be clarified if it must be removed or not.

Reply:
I removed refreshDelayInSecods Parameter in documentation.

I left the other parameter because it is still in use in OJElectronicsBridgeConfiguration.java